NSelene/SeleneElement.cs

```diff
@@ -330,40 +330,87 @@ ReadOnlyCollection<IWebElement> SeleneContext.FindElements (By by)
             Should(Be.Visible);
             return this.ActualWebElement.FindElements(by);
         }
+        /// <remarks>
+        ///     This method executes JavaScript in the context of the currently selected frame or window.
+        ///     This means that "document" will refer to the current document and "element" will refer to this element
+        ///  </remarks>
+        public void ExecuteScript(string script, params object[] args)
+        {
+            IJavaScriptExecutor js = (IJavaScriptExecutor)this.driver.Value;
+            js.ExecuteScript($@"
+                return (function(element, args) {{
+                    {script}
+                }})(arguments[0], arguments[1])", new object[] { this.ActualWebElement, args });
+        }
+
+        /// <remarks>
+        ///     This method executes JavaScript in the context of the currently selected frame or window.
+        ///     This means that "document" will refer to the current document and "element" will refer to 
+        ///     this element If the script has a return value, then the following steps will be taken:
+        ///     <list type="bullet">
+        ///     <item>For an HTML element, this method returns a OpenQA.Selenium.IWebElement</item> 
+        ///     <item>For a number, a System.Int64 is returned</item> 
+        ///     <item>For a boolean, a System.Boolean is returned</item> 
+        ///     <item>For all other cases a System.String is returned.</item> 
+        ///     <item>For an array,we check the first element, and attempt to return a System.Collections.Generic.List
+        ///     of that type, following the rules above. Nested lists are not supported.</item> 
+        ///     <item>If the value is null or there is no return value, null is returned.
+        ///     Arguments must be a number (which will be converted to a System.Int64), a System.Boolean,
+        ///     a System.String or a OpenQA.Selenium.IWebElement. An exception will be thrown
+        ///     if the arguments do not meet these criteria. The arguments will be made available
+        ///     to the JavaScript via the "arguments" magic variable, as if the function were
+        ///     called via "Function.apply"</item> 
+        ///     </list>
+        ///  </remarks>
+        public object GetFromExecuteScript(string script, params object[] args)
```

Comment:
I recommend to keep things simple... It's already kind of a standard that ExecuteScript already can return a value... no need to specify GetFrom prefix...